### PR TITLE
Replace foreman-ruby with tfm-ruby

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
Needs https://github.com/theforeman/foreman/pull/6680

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -1077,6 +1077,8 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 
 sed -i '/^ExecStart/a EnvironmentFile=-%{_sysconfdir}/sysconfig/%{name}' %{buildroot}%{_unitdir}/%{name}.service
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec]: Can we drop this one? In the installer I want to use systemd override files

File: packages/foreman/foreman/foreman.spec [hunk 2]
```diff
@@ -1077,6 +1077,8 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
...
+sed -i 's/\/usr\/bin\/rails/\/usr\/bin\/scl enable tfm "rails' %{buildroot}%{_unitdir}/%{name}.service
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec]: How about `sed -i '/^ExecStart/ s|/usr/bin/rails \(.\+\)$|/usr/bin/scl enable tfm "rails \1"|' %{buildroot}%{_unitdir}/%{name}.service`

**Comment by ekohl** [comment]
Could you have a look at my comments?

**Comment by ehelms** [comment]
Working to test this as we are now chicken-n-egg:

http://koji.katello.org/kojifiles/work/tasks/1405/191405/build.log

**Comment by ehelms** [comment]
Updated, local scratch build:

http://koji.katello.org/koji/taskinfo?taskID=191410

I verified this contains the right entry in the service file. Thanks for the sed @ekohl

**Pull request merged**